nuxt.config.js

```diff
@@ -27,7 +27,7 @@ export default {
         hid: 'description',
         name: 'description',
         content:
-          'Modrinth is a mod distribution platform. Modrinth is modern, easy to use, and built for modders. Modrinth currently supports Minecraft, including Forge and Fabric mod loaders.',
+          'Download Minecraft fabric and forge mods on Modrinth. Discover and publish projects on Modrinth with a modern, easy to use interface and API.',
```

Comment:
Make sure Fabric and Forge are capitalised
```suggestion
          'Download Minecraft Fabric and Forge mods on Modrinth. Discover and publish projects on Modrinth with a modern, easy to use interface and API.',
```